Updates to uuid types

jkeifer opened the pull request:
Currently converts `action_uuid` to v7 format with timestamp to allow sorting results by record creation time.

Switch to v5 for payload cache uuids is coming next.

jkeifer committed:
Use latest postgres debian image for Dockerfile

jkeifer committed:
switch action_uuid to use uuid v7 for ordering

jkeifer committed:
ensure fixture test db is dropped on error

jkeifer committed:
tests to ensure migrations apply

geomatician commented on tests/fixtures/migrations/0_base_01.sql:
```diff
@@ -0,0 +1,110 @@
+  'id4',
+  NULL
+);
+
```
I had deleted the SQL for the `input_item` and `item_payload` tables in my branch... why are we adding it back here?

geomatician commented on tests/fixtures/migrations/0_base_01.sql:
```diff
@@ -0,0 +1,110 @@
+INSERT INTO swoop.item_payload (item_uuid, payload_uuid) VALUES (
+  '01887754-5a7c-430f-abc7-3b4ac0b1281d',
+  'ade69fe7-1d7d-472e-9f36-7242cc2aca77'
+);
```
Same comment here- `item_payload` statements was deleted in my branch.

jkeifer committed:
test coverage of action uuid/created_at constraint

jkeifer committed:
drop payload_hash in favor of v5 payload_uuid

Turns out uuid v5 is what we were trying to get from payload_hash, and
allows us to use effectively the same hash in uuid form rather than
having to store an additional random uuid.

We can't perform a forward migration of existing payload_hash'es into v5
uuids, so the migration here truncates all table data (as actions relate
to payloads, everything must go). Typically this sort of change would
be a non-starter for a database migration, but given the early phase of
development for this project it seems like a reasonable action to take.

jkeifer committed:
CHANGELOG updates for release v7.0.0
